index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles/style.css" />
+  </head>
+  <body>
+    <nav>
+      <a href="#">Tuzy Orizu</a><a><img src="img/Icons/Union.png" alt="" /></a>
+    </nav>
```

Comment:
Good job by using the `nav` tag for navigation. Kindly wrap the `nav` tag with the `header` tag. 
#### For example: 
```
<header> <nav> ... </nav> </header>
```
to make it easy for search engines and screen readers to detect easily, what's the header and what it contains. 😃